staging/src/k8s.io/apiserver/pkg/server/options/audit_test.go

```diff
@@ -35,6 +42,14 @@ func TestAuditValidOptions(t *testing.T) {
 	webhookConfig := makeTmpWebhookConfig(t)
 	defer os.Remove(webhookConfig)
 
+	policy := makeTmpPolicy(t)
+	defer os.Remove(policy)
+
+	utilfeature.DefaultFeatureGate.SetFromMap(map[string]bool{string(features.DynamicAuditing): true})
```

Comment:
just noticed the global feature gate modifications in these unit tests. use the SetFeatureGateDuringTest helper (see elsewhere for usage) to ensure we always restore the original value… failing to do that because of an early exit failed test can lead to really squirrelly behavior elsewhere